frontend/src/components/App/runCommand.ts

```diff
@@ -25,7 +25,7 @@
  * ```
  */
 export function runCommand(
-  command: 'minikube' | 'az',
+  command: 'minikube' | 'az' | 'electron-node',
```

Comment:
I am not sure of the effectiveness of this command restriction right now. But having `electron-node` as a command will certainly allow us to run a lot of things that this list suggests we shouldn't.